gen/flights/flightplan.py

```diff
@@ -114,6 +116,65 @@ def tot_waypoint(self) -> Optional[FlightWaypoint]:
         failed to generate. Nevertheless, we have to defend against it.
         """
         raise NotImplementedError
+    
+    @cached_property
+    def bingo_fuel(self) -> int:
+        """Bingo fuel value for the FlightPlan
+        """
+        if self.waypoints is None or self.farthest_wpt_from_arrival is None:
+            return 0
+
+        distanceToArrival = self.farthest_wpt_from_arrival.position.distance_to_point(self.flight.arrival.position)
+        distanceToArrival = meter_to_nm(distanceToArrival)
+
+        bingo = 1000 # Minimum Emergency Fuel
+        bingo += 500 # Visual Traffic
+        bingo += 15 * distanceToArrival
+
+        if self.flight.divert is not None and self.farthest_wpt_from_divert is not None:
+            assert self.farthest_wpt_from_divert is not None
+            assert self.flight.divert is not None
```

Comment:
These are the condition, so I'm guessing this is to shut up the type checker? Instead, do:

```
divert = self.flight.divert
farthest_from_divert = self.farthest_wpt_from_divert
if divert is not None and farthest_from_divert is not None:
    ...
```

Reply:
Got another typechecking error:
```
        divert = self.flight.divert
        farthest_from_divert = self.farthest_wpt_from_divert
        if divert is not None and farthest_from_divert is not None:
            bingo += 10 * meter_to_nm(farthest_from_divert)

```

```
gen\flights\flightplan.py:124: error: Argument 1 to "meter_to_nm" has incompatible type "Optional[float]"; expected "float"
```

Reply:
My fault, its fixed.